data/nlu/responses/responses_spenden.yml

```
@@ -5,6 +5,12 @@ responses:
     - text: "Ich habe verstanden, dass es um das Thema Spenden geht. Um welche Spendenartikel handelt es sich?"
 
   utter_ask_spendeErhalten:
+    - condition:
+      - type: slot
+        name: hygieneartikel
+        value: [Tampon]
```

```suggestion
        value: hygieneartikel
```
Ich meine, dass das so funktioniert, da der slot `hygieneartikel` vom Type `list` ist

---

Ok vergiss es, ich hab grad gesehen, dass du das immer so hast. Hoffen wir mal, dass es auch mit anderen Hygieneartikeln funktioniert 😁